src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/AutoRerouteV2Spec.groovy

```diff
@@ -114,49 +117,13 @@ class AutoRerouteV2Spec extends HealthCheckSpecification {
         cleanup: "Restore topology to the original state, remove the flow"
         flowHelperV2.deleteFlow(flow.flowId)
         portDown && !portUp && antiflap.portUp(isl.dstSwitch.dpId, isl.dstPort)
-        broughtDownPorts.every { antiflap.portUp(it.switchId, it.portNo) }
-        Wrappers.wait(discoveryInterval + WAIT_OFFSET) {
+        broughtDownIsls.each { antiflap.portUp(it.srcSwitch.dpId, it.srcPort) }
+        wait(discoveryInterval + WAIT_OFFSET) {
             assert northbound.getActiveLinks().size() == topology.islsForActiveSwitches.size() * 2
         }
         database.resetCosts()
     }
 
-    @Tags([VIRTUAL, LOW_PRIORITY])
```

Comment:
It is failed isl that cause the flow to reroute, so the test makes no sense. Moreover, it is misleading, since switch disconnect itself does not cause a flow reroute (for example flow will be absolutely fine if round-trip remain active)